src/js/components/SitePageLoading.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import SiteHeader from './SiteHeader';
+
+import { Skeleton } from 'antd';
+
+const SitePageLoading = () => {
+  return (
+    <>
+      <SiteHeader />
```

Comment:
it doesn't make sense to render SiteHeader here based on the context of where it's used - it will just double-render SiteHeader